Area chart changes (#209)
patternfly/training-scenarios: react-charts/area-chart/intro.md

PatternFly React charts are based on the [Victory chart](https://formidable.com/open-source/victory/docs/victory-chart/) library, along with additional functionality, custom components, and theming for PatternFly.

PatternFly provides a collection of React-based components you can use to build patterns with consistent markup, styling, and behavior.

> PatternFly React charts are composed using separate components, so features are more easily customizable. In this course, you will build a PatternFly area chart together - starting with a simple chart, adding multiple datasets, tooltips, axis labels, a legend, and concluding by changing the theme color.

Reviewer: Grammar and use of you. "... you will build a PatternFly area chart together - ...".
```suggestion
PatternFly React charts are composed using separate components, so features are more easily customizable. This course covers area charts - starting with a simple chart, adding multiple datasets, tooltips, axis labels, a legend, and concluding by changing the theme color.
```

Reviewer 2: I like Jon's suggestion... I was questioning the use of 'together' while reviewing another module.  Another option could be to strike the word 'together' and leave the rest of the sentence as is.

Reviewer 3: Hi! I left feedback on "you" vs. "we" earlier (and in the Katacoda feedback doc I shared after user testing, https://docs.google.com/document/d/1srD1id97Pi_2cLCPGeemwX1aujc6mXMrcd8TCarXmi0/edit?usp=sharing). We never want to use "we." Instead, we should always use "you," or cut the pronoun altogether if possible. This puts focus on the user, delivers the info more efficiently, and prevents ambiguity/confusion from jumping between different points of view.
